cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/customaction/DefaultCustomActionConfigurer.java

```diff
@@ -42,7 +46,9 @@
   private String description;
   private Map<String, String> properties;
 
-  private DefaultCustomActionConfigurer(CustomAction customAction) {
+  public DefaultCustomActionConfigurer(CustomAction customAction, Id.Namespace deployNamespace, Id.Artifact artifactId,
```

Comment:
I don't see any `new DefaultCustomActionConfigurer` in that class. Do I miss something?